integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -35,8 +36,43 @@ window.addEventListener("load", () => {
       latitude.dataset.initial = latitude.value;
     });
   }
+
+  document.getElementById("map_preview_button").addEventListener("click", showPreview);
 });
 
+function showPreview(){
+  const script = document.createElement("script");
+  script.src = "https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js";
+  document.body.appendChild(script);
+
+  var lng = (document.getElementById("id_longitude") as HTMLInputElement).value;
+  var lat = (document.getElementById("id_latitude") as HTMLInputElement).value;
+
+  if (lng && lat){
+    var map = new maplibregl.Map({
+      container: 'map',
+      style: 'https://api.maptiler.com/maps/streets/style.json?key=get_your_own_OpIi9ZULNHzrESv6T2vL', // stylesheet location
```

Comment:
```suggestion
      style: 'https://maps.tuerantuer.org/styles/integreat/style.json',
```
Probably not a good idea to include the key in the source code.
Also maybe we can use our own tileserver https://maps.tuerantuer.org/?